src/krkn_lib/elastic/krkn_elastic.py

```diff
@@ -0,0 +1,251 @@
+
+        :param run_uuid: the krkn run id
+        :param raw_data: the mixed type dictionary (will be validated
+            checking the attributes types )
+        :param index: the elasticsearch index where
+            the object will be stored
+        :return: the time needed to save if succeeded -1 if failed
+        """
+        if not index:
+            raise Exception("index cannot be None or empty")
+        if not run_uuid:
+            raise Exception("run uuid cannot be None or empty")
+        time_start = time.time()
+        try:
+            for metric in raw_data:
+                if (
+                    "name" in metric
+                    and "values" in metric
+                    and isinstance(metric["name"], str)
+                    and isinstance(metric["values"], list)
+                ):
+                    values = [
+                        ElasticMetricValue(timestamp=v[0], value=v[1])
+                        for v in metric["values"]
+                        if isinstance(v[0], int) and isinstance(v[1], float)
+                    ]
+                    metric = ElasticMetric(
+                        run_uuid=run_uuid,
+                        name=metric["name"],
+                        values=values,
+                        created_at=datetime.datetime.now(),
+                    )
+                    result = self.push_metric(metric, index)
+                    if result == -1:
+                        self.safe_logger.error(
+                            f"failed to save metric "
+                            f"to elasticsearch : {metric}"
+                        )
+            return int(time.time() - time_start)
+        except Exception:
+            return -1
+
+    def push_alert(self, alert: ElasticAlert, index: str) -> int:
+        """
+        Pushes an ElasticAlert object to elastic
+
+        :param alert: the populated ElasticAlert object
+        :param index: the index where the ElasticAlert Object
+            is pushed
+        :return: the time needed to save if succeeded -1 if failed
+        """
+        if not index:
+            raise Exception("index cannot be None or empty")
+        try:
+            time_start = time.time()
+            alert.save(using=self.es, index=index)
+            return int(time.time() - time_start)
+        except Exception:
+            return -1
+
+    def push_metric(self, metric: ElasticMetric, index: str) -> int:
+        """
+        Pushes an ElasticMetric object to elastic
+
+        :param metric: the populated ElasticMetric object
+        :param index: the index where the ElasticMetric Object
+            is pushed
+        :return: the time needed to save if succeeded -1 if failed
+        """
+        if not index:
+            raise Exception("index cannot be None or empty")
+        try:
+            time_start = time.time()
+            metric.save(using=self.es, index=index)
```

Comment:
Tullio, what functionality does this do? I can't find save as a function under the ElasticMetric model and am a little confused 

Getting this error repeatedly though and not sure where its from
```
2024-07-19 14:16:50,370 [INFO] POST https://search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com:443/krkn_chaos_ci/_doc [status:406 duration:0.082s]
2024-07-19 14:16:50,370 [WARNING] Node <Urllib3HttpNode(https://search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com:443)> has failed for 1 times in a row, putting on 1 second timeout
2024-07-19 14:16:50,370 [INFO] telemetry collection disabled, skipping.
2024-07-19 14:16:50,371 [INFO] Alerts checking is enabled
2024-07-19 14:16:50,390 [ERROR] Failed to get the metrics: Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,391 [ERROR] Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,391 [ERROR] Failed to get the metrics: Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,391 [ERROR] Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,392 [ERROR] Failed to get the metrics: Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,392 [ERROR] Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,393 [ERROR] Failed to get the metrics: Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,393 [ERROR] Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,394 [ERROR] Failed to get the metrics: Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,394 [ERROR] Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,394 [ERROR] Failed to get the metrics: Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,394 [ERROR] Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
2024-07-19 14:16:50,395 [ERROR] Failed to get the metrics: Invalid URL '/api/v1/query_range': No scheme supplied. Perhaps you meant https:///api/v1/query_range?
```